tests/ui/lifetimes/temporary-lifetime-extension.rs

```diff
@@ -1,4 +1,10 @@
-//@ check-pass
+//@ run-pass
```

Comment:
Any reason this test needs to run?

Reply:
I forgot about the `//@ check-run-results`. I will add it later.

Reply:
I also want to make sure that the print-out makes sense.

Reply:
Applied. The print-out makes sense.